wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,76 @@
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{the OEM's name} drivers".
+ 2. Search for the model number of the piece of hardware you want to upgrade/install drivers for.
+ 3. When you select the product, you may have two options:
+	- It's recommended to use the driver downloader if it's provided, as it makes the installation a lot easier.
+	- Download this specific driver only and install manually.
+- See [Reinstalling GPU drivers from the OEM's website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information.
+## Troubleshooting 🔧
+
+1. Enter Safe Mode:
+
+Safe mode is a diagnostic tool, which loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+
+ - Windows 10/11: Go to your start menu and with shift pressed, click restart. But if you have issues, you can try to press repeatedly and hold the power button while booting to trigger a forced restart. After 2–3 restarts, you'll see the “Advanced options” screen. Navigate to “Troubleshoot” > “Advanced options” > “Startup Settings” > “Restart now”. Upon restart, choose “Enable Safe Mode with Networking” (if you need internet) or simply “Safe Mode”.
+
+ - Earlier Windows versions: Restart your computer and repeatedly press F8 before the Windows logo appears. This should open the Advanced Boot Options menu, where you can select “Safe Mode”.
+
+2. Uninstall the Driver in Safe Mode:
+
+Once in safe mode, you can uninstall the problematic driver:
+
+ 1. Open the Device Manager (search for it in the Start Menu).
+ 2. Expand the category for the device with the recently installed driver.
+ 3. Right-click on the device and select “Uninstall device”.
+ 4. Check the box “Delete the driver software for this device” (if available) and click “Uninstall”.
+
+3. System Restore (if applicable):
+
+If you recently created a system restore point before installing the driver, you can restore Windows to that point:
+
+ 1. In Safe Mode, open the Start Menu and search for “System Restore”.
+ 2. Click “Create a restore point” and then “Next”.
+ 3. Choose the desired restore point (ideally the one before installing the driver) and click “Next” followed by “Finish”.
+
+4. Startup Repair (if Windows won't boot at all):
+
+If Safe Mode isn't accessible, you can try Startup Repair, a built-in tool on Windows that will attempt to fix boot issues:
+
+ 1. Create a Windows Recovery USB drive using another computer. See : [Download Windows](https://msft.chat/wiki/downloading-windows.html).
+ 2. Boot your computer using the recovery drive and pick “Fix computer problems > Troubleshoot” > “Advanced options” > “Startup Repair”.
+
+5. Reinstall Windows (last resort):
+
+If none of the above solutions work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup.
```

Comment:
Relative link:
```suggestion
If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You may learn how to backup your data [here](/wiki/backup.html).
```